File: Python/백준/Bronze/7510. 고급 수학/고급 수학.py
t = int(input())
for i in range(t):
    aa = list(map(int,input().split()))
    aa.sort()
    a = aa[0]; b = aa[1]; c = aa[2]
    print(f'Scenario #{i+1}:')
    if (c**2==a**2+b**2):
        print('yes\n')
    else:
        print('no\n')

File: C#/백준/Bronze/7510. 고급 수학/고급 수학.cs
using System;
using System.Linq;

namespace q{
    class w{
        static void Main(String[] args){
            int t = int.Parse(Console.ReadLine());
            for (int i=0;i<t;i++){
                int[] aa = Array.ConvertAll(Console.ReadLine().Split(),int.Parse);
                int total = aa.Min()*aa.Min()+(aa.Sum()-aa.Max()-aa.Min())*(aa.Sum()-aa.Max()-aa.Min());
                if(aa.Max()*aa.Max()==total){
                    Console.WriteLine($"Scenario #{i+1}:\nyes\n");
                }
                else{
                    Console.WriteLine($"Scenario #{i+1}:\nno\n");
                }
            }
        }
    }
}